### ISSUE TITLE
Sentry API calls should be guarded against network errors.

### ISSUE THREAD

**Issue by rtyler** [CodeValet/canary#2]
https://sentry.io/codevalet/canary/issues/342771581/

```
OpenSSL::SSL::SSLError: SSL_connect SYSCALL returned=5 errno=0 state=SSLv2/v3 read server hello A
  app/../views/index.haml:49:in `block (2 levels) in singleton class'
    - events = SentryApi.project_issues(project.slug)
  app/../views/index.haml:46:in `block in singleton class'
    - projects.auto_paginate do |project|
  app/canary.rb:34:in `block in <class:App>'
    haml :index,
...
(74 additional frame(s) were not displayed)

OpenSSL::SSL::SSLError: SSL_connect SYSCALL returned=5 errno=0 state=SSLv2/v3 read server hello A
```

**Comment by rtyler** [comment]
Fixed in #7

**Comment by rtyler** [comment]
https://sentry.io/codevalet/canary/issues/343319621/
